firebase-client/index.js

```diff
@@ -0,0 +1,26 @@
+var google = require('googleapis');
+
+var config = require('config.js');
```

Comment:
Hi this refers to [Note](https://github.com/systemaccounting/mxfactorial/pull/50/files#r70177293)